src/actors/redis-scanner/redis-scanner.py

```diff
@@ -0,0 +1,44 @@
+import os
+import re
+import json
+import shlex
+import subprocess
+
+
+def _execute(cmd):
+    out = subprocess.check_output(shlex.split(cmd))
+    return out.strip()
+
+
+def get_version():
+    out = _execute('redis-server --version')
+    m = re.match(r'Redis server v=(.+?) +', out)
+    return m.group(1) if m else None
+
+
+def get_db_path():
+    out_dir = _execute('redis-cli -h localhost config get dir')
+    out_file = _execute('redis-cli -h localhost config get dbfilename')
+
+    try:
+        parsed_dir = out_dir.split()[1]
+        parsed_file = out_file.split()[1]
+    except (TypeError, IndexError):
```

Comment:
In that case I think ```TypeError``` will not be raised here. _execute method will raise different exception, in worst case you will have ```IndexError``` or exception from _execute.

Reply:
That's right, thanks!